### DISCUSSION TITLE
Error for setup.py

### DISCUSSION THREAD

**Discussion by Rami-Ismael** [OpenMined/courses#340] [Course: Our Privacy Opportunity]
I was download (pip install git+https://github.com/OpenMined/PySyft@dev#egg=syft). However, an error pop up. The error was this

ERROR: File "setup.py" or "setup.cfg" not found for legacy project syft from git+https://github.com/OpenMined/PySyft@dev#egg=syft.

**Comment by TianyueChu** [comment]
I have this ERROR too

**Comment by znreza** [comment]
Hi @Rami-Ismael could you please try this way that is suggested [here](https://github.com/OpenMined/courses/discussions/307#discussioncomment-839323)? You can now directly install [syft 0.5](https://github.com/OpenMined/PySyft/tree/syft_0.5.0) as well.
